bats/sql-merge.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+
+    dolt sql <<SQL
+CREATE TABLE test (
+    pk int primary key
+);
+
+INSERT INTO test VALUES (0),(1),(2);
+SQL
+}
+
+teardown() {
+    teardown_common
+}
+
+@test "DOLT_MERGE works with ff" {
+        dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Step 1');
+SELECT DOLT_CHECKOUT('-b', 'feature-branch');
+INSERT INTO test VALUES (3);
+SELECT DOLT_COMMIT('-a', '-m', 'this is a ff');
+SELECT DOLT_CHECKOUT('master');
+SQL
+    run dolt sql -q "SELECT DOLT_MERGE('feature-branch');"
+    [ $status -eq 0 ]
+
+    run dolt log -n 1
+    [ $status -eq 0 ]
+    [[ "$output" =~ "this is a ff" ]] || false
+
+    run dolt sql -q "SELECT COUNT(*) FROM dolt_log"
+    [ $status -eq 0 ]
+    [[ "$output" =~ "3" ]] || false
+}
+
+@test "DOLT_MERGE correctly returns head and working session variables." {
```

Comment:
Need at least one test of error conditions

Reply:
And a test of the --no-ff option

Reply:
Did not implement no-ff yet